app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -325,38 +325,35 @@ fun CreateActivityScreen(
                               Toast.LENGTH_SHORT)
                           .show()
                       return@Button
-                    } else if (attendees.size >= placesMax.toInt()) {
-                      Toast.makeText(
-                              context,
-                              context.getString(R.string.max_places_exceed),
-                              Toast.LENGTH_SHORT)
-                          .show()
-                      return@Button
                     } else if (creator == "") {
                       Toast.makeText(
                               context,
                               context.getString(R.string.login_check_in_create),
                               Toast.LENGTH_SHORT)
                           .show()
                       return@Button
-                    } else if (!hourDateViewModel.isBeginGreaterThanEnd(startTime, duration)) {
```

Comment:
I don't really understand why you remove dthis condition.

Reply:
This condition had to be removed because we don't have an "end time" anymore, it was transformed into "duration". So basically, if I set start time at 7:00 and duration of activity is 2h, then this check would wrongly give the toast "start time must be before end time".

Let me know if it's more clear now, but it was necessary to remove this logic.

Reply:
OK now I understand